src/Attributes/MapFrom.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Spatie\DataTransferObject\Attributes;
+
+use Attribute;
+use Spatie\DataTransferObject\Caster;
+use Spatie\DataTransferObject\Exceptions\InvalidCasterClass;
+
+#[Attribute(Attribute::TARGET_CLASS | Attribute::TARGET_PROPERTY)]
+class MapFrom
+{
+    public function __construct(
+        protected mixed $attribute,
```

Comment:
Maybe narrow this down, to be more explicit in which values are accepted?
```suggestion
        protected int | string $attribute,
```

Reply:
Thanks for your feedback @abenerd! I just made the change.